app/assets/src/components/ui/icons/InfoSIcon.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const InfoSIcon = props => {
```

Comment:
what does the 'S' in InfoSIcon standfor?

Reply:
I believe it stands for "solid," since we have a similar icon that isn't filled in. @happyimadesignr ?

Reply:
Oh hey, it's for small. Sorry it's not clear now b/c it's a part of the new design system work. We're creating different sized icons, the smaller one will be solid and larger ones will be line icons!

Reply:
I'd prefer then to use the whole word "InfoSolidIcon"

Reply:
Renamed it to InfoSmallIcon!